### PR TITLE
Create a `<SummaryPage />` component

### PR DESCRIPTION
**Pull request by Andresbl123**
The `<SummaryPage />` component is created, which is intended to show the summary of the steps of an assisted.

### CODE REVIEW

File: src/components/layout/Divider/Divider.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Divider, IDividerProps } from ".";
+
+const story = {
+  title: "layouts/Divider",
+  component: [Divider],
```
Comment (LinconOssa) [inline src/components/layout/Divider/Divider.stories.tsx:5]: Igual en todos los que tenga así de las dos observaciones .

File: src/components/data/BoxAttirbute/BoxAttirbute.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { BoxAttribute, IBoxAttributeProps } from ".";
+
+const story = {
+  title: "data/BoxAttribute",
+  component: [BoxAttribute],
```
Comment (LinconOssa) [inline src/components/data/BoxAttirbute/BoxAttirbute.stories.tsx:5]: Igual aquí quitar[] ya que estamos haciendo el cambio como usted nos comento.

File: src/components/data/BoxAttirbute/BoxAttirbute.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { BoxAttribute, IBoxAttributeProps } from ".";
+
+const story = {
```
Comment (LinconOssa) [inline src/components/data/BoxAttirbute/BoxAttirbute.stories.tsx:3]: Agregar esto ya que se esta haciendo el cambio para todas
const story: Meta<typeof BoxAttribute>

File: src/components/data/Accordion/stories/Accordion.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+};
+
+export type MockDataSummaryPage = typeof mockDataSummaryPage;
+
+const mockDataSummaryPage = [
+  {
+    section: "Información general",
+    attributes: [
+      {
+        attribute: "Nombre del casi de uso",
```
Comment (LinconOssa) [inline src/components/data/Accordion/stories/Accordion.stories.tsx:33]: Es "caso" de uso

**Review by LinconOssa — CHANGES_REQUESTED** [review]

File: src/components/data/Accordion/stories/Accordion.stories.tsx (hunk 4 above)
Comment (Andresbl123) [reply src/components/data/Accordion/stories/Accordion.stories.tsx:33]: resuelto

File: src/components/layout/Divider/Divider.stories.tsx (hunk 1 above)
Comment (Andresbl123) [reply src/components/layout/Divider/Divider.stories.tsx]: resuelto

File: src/components/data/BoxAttirbute/BoxAttirbute.stories.tsx (hunk 2 above)
Comment (Andresbl123) [reply src/components/data/BoxAttirbute/BoxAttirbute.stories.tsx]: resuelto

File: src/components/data/BoxAttirbute/BoxAttirbute.stories.tsx (hunk 3 above)
Comment (Andresbl123) [reply src/components/data/BoxAttirbute/BoxAttirbute.stories.tsx]: resuelto

**Review by LinconOssa — APPROVED** [review]
Cumple requerimiento

File: src/components/data/Accordion/stories/ControllerAccordion.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
+import { MdArrowBack } from "react-icons/md";
```
Comment (carin91) [inline src/components/data/Accordion/stories/ControllerAccordion.tsx:1]: This should be a Layout, we need to create several summary pages (users, roles, LinixUseCases), therefore, there should be a component to do that.

I will assign a new card for this component, meanwhile, delete file. Could you reuse some of this code for the component? If so, save it for you.

However, create a Story for the purpose of this task.

image

File: src/components/data/Accordion/stories/Accordion.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+import {
+  ControllerAccordion,
+  IControllerAccordionProps,
+} from "./ControllerAccordion";
+
+import { parameters, props } from "./props";
+import { Accordion, IAccordionProps } from "..";
+
+const meta: Meta<IAccordionProps> = {
```
Comment (carin91) [inline src/components/data/Accordion/stories/Accordion.stories.tsx:12]: The meta element with a type of Meta requires the component as argument, not the component props. Therefore, this type should be changed considering the type of Accordion

image

File: src/components/data/Accordion/stories/Accordion.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+  title: "data/Accordion",
```
Comment (carin91) [inline src/components/data/Accordion/stories/Accordion.stories.tsx:13]: I would use a different path. In the project folder you placed this component into components/data/Accordion, why do you use a different folder in the storybook? 

image

File: src/components/data/Accordion/styles.ts [hunk 8]
```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import { inube } from "@inube/design-system";
+
+interface IStyledContainer {
+  isMobile: boolean;
+}
+
+export const StyledContainer = styled.div<IStyledContainer>`
```
Comment (carin91) [inline src/components/data/Accordion/styles.ts:8]: Why do you need to create a styled.div, i would use the Stack component from the inube design-system, could you not?

File: src/components/data/Accordion/styles.ts [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+  display: flex;
+  padding: ${({ isMobile }) => (isMobile ? "16px" : "20px")};
```
Comment (carin91) [inline src/components/data/Accordion/styles.ts:10]: We must use the inube.spacing, even with 0, refactor to use the spacing tokens.

File: src/components/data/Accordion/styles.ts [hunk 10]
```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import { inube } from "@inube/design-system";
+
+interface IStyledContainer {
+  isMobile: boolean;
+}
+
+export const StyledContainer = styled.div<IStyledContainer>`
+  display: flex;
```
Comment (carin91) [inline src/components/data/Accordion/styles.ts:9]: This is the default definition for flex-direction, is it not? why do you need to assign the property?

File: src/components/data/Accordion/styles.ts [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+  padding: ${({ isMobile }) => (isMobile ? "16px" : "20px")};
+  flex-direction: column;
+  gap: "16px";
```
Comment (carin91) [inline src/components/data/Accordion/styles.ts:12]: We must use the inube.spacing, even with 0, refactor to use the spacing tokens.

File: src/components/data/Accordion/styles.ts [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+  border-radius: 8px;
+  width: auto;
+  min-height: 160px;
```
Comment (carin91) [inline src/components/data/Accordion/styles.ts:15]: This property assignment should be deleted/refactored. It breaks the expected behavior of setIsOpen

Comment (carin91) [reply src/components/data/Accordion/styles.ts:15]: image

File: src/components/data/Accordion/styles.ts [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+  box-sizing: border-box;
+  align-items: flex-start;
+  border: 1px solid
```
Comment (carin91) [inline src/components/data/Accordion/styles.ts:18]: We must use the inube.spacing, even with 0, refactor to use the spacing tokens.

File: src/components/data/Accordion/styles.ts [hunk 14]
```diff
@@ -0,0 +1,33 @@
...
+    ${({ theme }) =>
+      theme.color?.stroke?.divider?.regular ||
+      inube.color.stroke.divider.regular};
+`;
+
+export const StyledHead = styled.button`
```
Comment (carin91) [inline src/components/data/Accordion/styles.ts:24]: I strongly disagree with this. Are you using a button as the container of the whole component? Refactor this to use the proper html elements. I strongly suggest to use the Stack component from inube design system

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/components/layout/Divider/Divider.stories.tsx [hunk 15]
```diff
@@ -0,0 +1,14 @@
+import { Meta } from "@storybook/react";
+import { Divider, IDividerProps } from ".";
+
+const meta: Meta<IDividerProps> = {
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/components/layout/Divider/Divider.stories.tsx:4]: Resuelto.

Aunque en este caso es lo mismo, ya que lo que hace Storybook es tomar el tipado del componente, que a la final es la `interface` 
image
image

File: src/components/data/Accordion/styles.ts (hunk 13 above)
Comment (Andresbl123) [reply src/components/data/Accordion/styles.ts:19]: aquí pienso que no se deben usar los token de espaciado por que a la final están pensado mara las margins,  padding y el gap, no para bordes o definición de tamaños

File: src/components/data/Accordion/styles.ts (hunk 11 above)
Comment (Andresbl123) [reply src/components/data/Accordion/styles.ts:13]: resuelto

File: src/components/data/Accordion/styles.ts (hunk 10 above)
Comment (Andresbl123) [reply src/components/data/Accordion/styles.ts:10]: No entiendo muy bien, la sugerencia, pero el valor por default de `flex-direction` es `row`, por lo que aquí se cambia a `column`

File: src/components/layout/Divider/styles.ts [hunk 16]
```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+import { inube } from "@inube/design-system";
+import { IDividerProps } from ".";
+
+export const StyledDivider = styled.hr<IDividerProps>`
+  margin: 0;
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/components/layout/Divider/styles.ts]: para el height, si aplica, para las márgenes, se establece el valor en 0px, ya que si no se crean unas márgenes que afectan el maquetado, que podría ser por la herencia en css

[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/components/layout/Divider/styles.ts]: resuelto

File: src/components/layout/Divider/styles.ts [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+  width: 100%;
+  height: 0px;
```
[the remark this answers is not in the document]
Comment (Andresbl123) [reply src/components/layout/Divider/styles.ts]: resuelto

File: src/components/data/Accordion/styles.ts (hunk 12 above)
Comment (Andresbl123) [reply src/components/data/Accordion/styles.ts:16]: resuelto

File: src/components/data/Accordion/styles.ts (hunk 14 above)
Comment (Andresbl123) [reply src/components/data/Accordion/styles.ts:25]: resuelto

**Review by carin91 — APPROVED** [review]

**Pull request merged**